karoo_gp/population.py

```diff
@@ -186,8 +209,8 @@ def fitness_gene_pool(self, swim='p', tree_depth_min=None, terminals=None):
             elif swim == 'f':
                 # each tree must contain at least one instance of each feature
                 saved = tree.save()
-                missing = sum([1 for t in terminals.get()
-                               if f'({t.symbol})' not in saved])
+                missing = sum([1 for t in self.model.get_nodes(('terminal'))
+                               if f'({t.label})' not in saved])
```

Comment:
This also looks wrong:
```suggestion
                missing = sum(1 for t in self.model.get_nodes(['terminal'])
                               if f'({t.label})' not in saved)
```